CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+SystemJS 6.12.2
```

Comment:
Changelog in this project has usually been created by the person releasing the code